mlflow/utils/autologging_utils.py

```diff
@@ -614,50 +638,62 @@ def safe_patch_function(*args, **kwargs):
         # Whether or not an exception was raised from within the original / underlying function
         # during the execution of patched code
         failed_during_original = False
+        # Whether or not to exclude auto-autologged content from content explicitly logged via
+        # `mlflow.start_run()`
+        exclusive = get_autologging_config(autologging_integration, "exclusive", False)
 
-        try:
+        active_run = mlflow.active_run()
+
+        if active_run and exclusive and not _AutologgingSessionManager.active_session():
+            return original(*args, **kwargs)
```

Comment:
Should we move this up next to the disablement check on lines 628 / 629?

Reply:
Done!